src/Microsoft.DotNet.Build.Tasks.Packaging/src.Desktop/project.json

```diff
@@ -4,7 +4,7 @@
     "Newtonsoft.Json": "7.0.1",
```

Comment:
If I change this *and* change the framework to `net46` it works, but with `net45` it doesn't get the `Microsoft.NETCore.Targets.NETFramework` dependency. Is 4.6 vs. 4.5 a problem?